report-job-status/action.yml

```diff
@@ -79,8 +79,8 @@ runs:
             failedStepsMsg += failedSteps
           }
           
-          let message = `🔴 Job failed at <a href="${baseUrl}/${{ github.repository }}/">${{ github.repository }}</a>:
-          <b>Job</b>: <a href="${baseUrl}/${{ github.repository }}/actions/runs/${{ github.run_id }}">${{ github.job }}</a>
+          let message = `🔴 Job failed: <a href="${baseUrl}/${{ github.repository }}/actions/runs/${{ github.run_id }}">${{ github.job }}/${{ github.run_id }}</a>
```

Comment:
1. Looks like the title should be bold as well.
2. Why do you need `run ID` in the job name?

Reply:
1. I don't see why anything at all should be bold. Bold text can draw attention to the most important thing. When half of the message is in bold, it's just the same as nothing is in bold.
2. To make such messages searchable in the chat. Maybe ID should go somewhere below and not in the title.

Reply:
1. It should be bold because it is the most important thing in this message and it is strange that everything is bold except the job part.
2. Actually, first, we need to fix the bug in this action. The link to the job points to the workflow run, not job run.

For me the correct message should be something like this:
```
🔴 FAIL:
Job: test-job                                        # points to job run
Job ID: #1234567                                     # points to job run
Workflow run: test-workflow                          # points to workflow run
Workflow run ID: #7654321                            # points to workflow run
Repo: tarantool/tarantool
Pull request: #3086
Commit: ca76989c38db847a9c9da42ae50d36b7a78041cd
History: commits
Triggered on: pull_request
Committer: foobar
```

Reply:
done